Step01/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
```

Comment:
@kwoolytech please take the standard .gitignore file for android here as there are some files that are not required to be shared.

https://github.com/github/gitignore/blob/master/Android.gitignore